os/board/common/Kconfig

```diff
@@ -66,6 +66,12 @@ config ARCH_BOARD_HAVE_SECOND_FLASH
 if ARCH_BOARD_HAVE_SECOND_FLASH && (NFILE_DESCRIPTORS != 0)
 comment "Second Flash Partition Options"
 
+config FLASH_SECOND_START_ADDR
```

Comment:
Other configs are named with `SECOND_FLASH_XXXX`. Let's use the same format.

Reply:
done